pkg/csvparser/reader.go

```diff
@@ -113,6 +113,9 @@ type Reader struct {
 	// or the Unicode replacement character (0xFFFD).
 	Comma rune
 
+	// Quote is the single character used for marking fields limits
+	Quote []rune
```

Comment:
If it is a single character why is a slice?

Reply:
> If it is a single character why is a slice?

The reason is that 'Quote' can be empty. It means do not consider any quote while parsing